lib/phoenix/pubsub/local.ex

```diff
@@ -82,30 +88,35 @@ defmodule Phoenix.PubSub.Local do
   ## Examples
 
       iex> subscribers(:local_server, "foo")
-      Enumerable.t
+      [#PID<0.48.0>, #PID<0.49.0>]
 
   """
-  def subscribers(local_server, topic) do
-    GenServer.call(local_server, {:subscribers, topic})
+  def subscribers(local_server, topic) when is_atom(local_server) do
+    try do
+      :ets.lookup_element(local_server, topic, 2)
+    catch
+      :error, :badarg -> []
+    end
   end
 
   @doc false
-  def list(local_server) do
-    GenServer.call(local_server, :list)
+  # This is an expensive and private operation. DO NOT USE IT IN PROD.
+  def list(local_server) when is_atom(local_server) do
```

Comment:
We need it for tests. It is already `@doc false`, it is just a warning for those two decide to read the source and not respect the fact it is private.